views/pulsar/v2/helpers/html.html.twig

```diff
@@ -1581,13 +1581,13 @@ Datatable - https://jadu.github.io/pulsar/components/datatable/
                     {% for column in options.data.columns %}
                         {% if loop.first %}
                             {% if options['data-select'] is not defined or (options['data-select'] is defined and options['data-select'] != 'false') %}
-                                <th class="table-selection" data-orderable="false">
+                                <th class="table-selection" data-orderable="false" scope="col">
```

Comment:
Really nitpick, but can this be at the start like the other helper, or at least before the data attribute?